src/renderer/components/+entity-settings/entity-settings.module.css

```diff
@@ -19,26 +19,12 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-.EntitySettings {
-  $spacing: $padding * 3;
-
-
-  // TODO: move sub-component styles to separate files
-  .admin-note {
-    font-size: small;
-    opacity: 0.5;
-    margin-left: $margin;
-  }
-
-  .button-area {
-    margin-top: $margin * 2;
-  }
-
-  .file-loader {
-    margin-top: $margin * 2;
-  }
-
-  .Input, .Select {
-    margin-top: $padding;
-  }
-}
+.entityName {
+  @apply font-bold overflow-hidden;
```

Comment:
Should we stop using `@apply` directives as we going to get rid of `tailwindcss` in favour css-modules only?

Reply:
It's not yet decided, let's keep it for now in this PR.